watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -0,0 +1,329 @@
+###############################################################################
```

Comment:
Minor mod: I think this was an oversight for the metab multi_sweep file, but we should have the copyright header in your dye_sweep file.

Reply:
Thanks for catching that!